mmcv/cnn/bricks/activation.py

```diff
@@ -15,7 +15,19 @@
     ACTIVATION_LAYERS.register_module(module=module)
 
 if digit_version(torch.__version__) >= digit_version('1.7.0'):
-    ACTIVATION_LAYERS.register_module(module=nn.SiLU)
+    ACTIVATION_LAYERS.register_module(module=nn.SiLU, name='SiLU')
+else:
+
+    class SiLU(nn.Module):
+        """Sigmoid Weighted Liner Unit."""
+
+        def __init__(self, inplace=True):
+            super().__init__()
```

Comment:
```suggestion
        def __init__(self, inplace=True):
            super().__init__()
            self.inplace = inplace
```

Reply:
Should this `inplace` argument have a same default value with pytorch which sets `inplace=False` by default.